docs/clients/mpd.rst

```diff
@@ -171,5 +167,5 @@ projects are a real match made in heaven."
 Partify
 -------
 
-`Partify <http://www.partify.us/>`_ is a web based MPD client focusing on
+`Partify <http://partify.club/>`_ is a web based MPD client focusing on
```

Comment:
Mmyeah, that sounds reasonable, I'll get to it tonight probably (Sweden time).
On Wed, Oct 21, 2015 at 11:50:17PM -0700, Stein Magnus Jodal wrote:
> > @@ -171,5 +167,5 @@ projects are a real match made in heaven."
> >  Partify
> >  -------
> >  
> > -`Partify <http://www.partify.us/>`_ is a web based MPD client focusing on
> > +`Partify <http://partify.club/>`_ is a web based MPD client focusing on
> 
> Maybe link to https://github.com/fhats/partify instead?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mopidy/mopidy/pull/1317/files#r42715997